h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -510,7 +510,16 @@ public static final TwoDimTable makeTwoDimTable(String tableHeader, String sort_
     String[] rowHeaders = new String[length];
     for (int i = 0; i < length; i++) rowHeaders[i] = "" + i;
 
-    if ("mean_per_class_error".equals(sort_metric)){ //Multinomial
+    if (sort_metric == null && length == 0) {
```

Comment:
The AutoML process can use up all its time before the first model is built.  I saw it happen during PR tests on a very heavily loaded Jenkins machine.  You can force it to happen by setting the max_time_secs to 0.01.